uwebsockets/client.py

```diff
@@ -33,10 +33,12 @@ def connect(uri):
     sock = socket.socket()
     addr = socket.getaddrinfo(uri.hostname, uri.port)
     sock.connect(addr[0][4])
+    if uri.port == 443:
+        sock = ussl.wrap_socket(sock)
```

Comment:
I would rather trigger this off protocol, `if protocol == 'wss'` that way it's possible to do SSL on non-443 ports.

Reply:
That sounds sensible. What about the case where the protocol is omitted but the provided port is 443 ? Do we just assume the request is for a non-encrypted connection on port 443 then ?

Reply:
That sounds sensible. What about the case where the protocol is omitted but the provided port is 443 ? Do we just assume the request is for a non-encrypted connection on port 443 then ?

Reply:
I think you should have to specify the protocol.